tensorflow/tools/ci_build/install/install_deb_packages.sh

```diff
@@ -38,3 +38,5 @@ apt-get install -y \
     zlib1g-dev
 apt-get clean
 rm -rf /var/lib/apt/lists/*
+
+pip install virtualenv
```

Comment:
Would python-virtualenv from apt-get work? It would be better to use the linux distro version if possible.